README.rst

```diff
@@ -62,7 +62,7 @@ If so, then **mirakuru** is what you need.
 that it's running. There are three basic executors with predefined conditions:
 
 
-* Executor - simply starts
+* SimpleExecutor - starts and doesn't wait for anyting. It's usefull to stop/kill its subprocesses.
```

Comment:
`Executor` seems to be more "internal" and I wouldn't hit library users with such detail in the beginning. I could add a separate section encouraging to implement own executor inheriting from `Executor`...